src/components/common/TargetCreationTab/styles.scss

```diff
@@ -0,0 +1,48 @@
+@import 'styles/mixins';
+@import 'styles/variables';
+
+.create-title {
```

Comment:
Here you can nest classes in order not to repeat so much code. It would go like this: 


```
create{
   &-title{
   ...
   }

   &-subtitle{
   ...
   }

   &-target{
   ...
   }
}

```